en/flight_controller/betafpv_beta75x.md

```diff
@@ -0,0 +1,33 @@
+# BetaFPV Beta75X 2S Brushless Whoop
+
+The [BetaFPV Beta75X](https://betafpv.com/products/beta75x-2s-whoop-quadcopter) is a very small quadrotor that can be flown indoors or outdoors, FPV or line-of-sight.
+
+![BetaFPV Beta75X](../../assets/hardware/betafpv_beta75x.jpg)
+
+## Where to Buy
+
+The *Beta75X* can be bought from a number of vendors, including:
+- [GetFPV](https://www.getfpv.com/beta75x-2s-brushless-whoop-micro-quadcopter-xt30-frsky.html)
+- [Amazon](https://www.amazon.com/BETAFPV-Beta75X-Brushless-Quadcopter-Smartaudio/dp/B07H86XSPW)
+
+In addition you will need an RC transmitter (make sure to select the matching receiver version), FPV goggles if you want to fly FPV, and a LiPo battery charger.
```

Comment:
- Can PX4 work with all the supplied/supported receiver types?
- We don't say anything about FPV in any docs, so I presume this is completely independent. Is there a "standard" for FPV goggles. Can we point to some options?
- LiPo batteries also needed.

Reply:
> Can PX4 work with all the supplied/supported receiver types?

Yes.

> We don't say anything about FPV in any docs, so I presume this is completely independent. Is there a "standard" for FPV goggles. Can we point to some options?

Yes it's independent. It's a bigger topic in itself, but I think the most commonly used are from Fat Shark (e.g. https://www.fatshark.com/product/dominator-hd3-core-fpv-goggles/).

> LiPo batteries also needed.

One set is included, but adding spare ones is a good idea.